doc/index.md

```diff
@@ -169,135 +57,6 @@ find and subscribe to those at: <https://lists.linuxcontainers.org>
 ### IRC
 If you prefer live discussions, you can find us in [#lxc](https://kiwiirc.com/client/irc.libera.chat/#lxc) on irc.libera.chat.
 
-## FAQ
```

Comment:
Moved this content to the existing FAQ page - will need to add a redirect (how is this currently done?).